lib/app_profiler/backend/vernier.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+begin
+  gem("vernier", ">= 0.4.0")
+  require "vernier"
+rescue LoadError
```

Comment:
I don't think this is necessary. Rails doesn't add extra warnings when it uses this pattern and I don't think we need to either.